swift/test/Ice/admin/TestI.swift

```diff
@@ -60,13 +68,11 @@ class RemoteCommunicatorI: RemoteCommunicator {
     }
 
     func updated(changes: [String: String]) {
```

Comment:
Could we make RemoteCommunicatorI an actor?

Reply:
I don't think it helps here.